otel-collector-config.yaml

```diff
@@ -26,4 +30,4 @@ service:
       receivers: [otlp]
       processors: [batch]
       exporters: [otlp/hny, logging]
-    #   exporters: [otlp/hny, logging, file]
+#   exporters: [otlp/hny, logging, file]

```

Comment:
I think this is here to demonstrate that you can also export to a file if you want to. I don't have a strong opinion about whether it should stay or not.

Reply:
I believe this PR only had the whitespace change, not the actual addition of the file exporter; I had added this previously, and there's a commented section above for adding the file exporter. This seemed relevant at the time since this repo is primarily used for local testing purposes, and having a small detail about how to quickly add the file exporter seemed helpful. I'm in favor of keeping it but can be convinced otherwise; either way it's probably separate from this PR 👍